pkg/utils/utils.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"flag"
+	log "github.com/Sirupsen/logrus"
```

Comment:
Ok, SGTM but we'll need to update soon.  libcalico is going to change to that import very soon, but so long as we're pinned to v1.5.1 then we're OK.